spring-amqp/src/main/java/org/springframework/amqp/support/converter/Jackson2JsonMessageConverter.java

```diff
@@ -53,23 +53,36 @@
 	private boolean typeMapperSet;
 
 	/**
+	 * Construct with an internal {@link ObjectMapper} instance and trusted packed to all ({@code *}).
+	 * @since 1.6.11
+	 */
+	public Jackson2JsonMessageConverter() {
+		this("*");
+	}
```

Comment:
Yeah... OK. I just thought that nobody use Jackson 1.x any more.
But right, backward compatibility process doesn't imply that.

Do you need commit on the matter or will do that on merge/cherry-picking?

I think `DefaultJackson2JavaTypeMapper` suffers the same condition.

Thanks